query-compiler/query-compiler/tests/snapshots/queries__queries@create-m2m.json.snap

```diff
@@ -0,0 +1,38 @@
+---
+source: query-compiler/query-compiler/tests/queries.rs
+expression: pretty
+input_file: query-compiler/query-compiler/tests/data/create-m2m.json
+---
+let 0 = unique (query «INSERT INTO "public"."User" ("email") VALUES ($1)
```

Comment:
Unrelated to this PR but I'm only now realizing we're missing the check if the query graph is transactional, and the corresponding query plan node. Should be easy to add.